orion/controllers/programs/programs_test.go

```diff
@@ -12,8 +12,9 @@ func TestExample(t *testing.T) {
 }
 
 func TestCheckProgram(t *testing.T) {
-  isValid := CheckValidProgram()
-  if (isValid == false) {
-    t.Errorf("Check was incorrect, got: %t, expected: %t.", isValid, false)
+  program := Program{Name: "Test Program", Grade1: 1, Grade2: 12}
+  isValid := CheckValidProgram(program)
+  if !isValid == false {
+    t.Errorf("Check was incorrect, got: %t, expected: %t.", isValid, true)
```

Comment:
Can we get more unit tests?
You've tested two words. What if it was three words?
What if there was one word?
Can you also test failure cases as well?